diagnosis_api/controllers/search_disease_by_syptoms.py

```diff
@@ -32,37 +32,44 @@ def search_disease_by_symptoms(self):
         """
 
         # check if the file exists
-        basepath = 'diagnosis_api/dataset/'
+        basepath = "diagnosis_api/dataset/"
```

Comment:
```suggestion
        basepath = "diagnosis-api/dataset/"
```
It's preferable and highly advised to use the hyphen instead of the lodash or underscore in urls

Reply:
the underscore in this case is due to the fact that diagnosis_api is a package, and packages can only have underscores.